styles/mixins.scss

```diff
@@ -70,69 +72,92 @@
     }
 }
 
+$-pin-mixin-variables-ns: --_--g-pin-mixin-border-radius;
```

Comment:
```suggestion
$pin-variables-ns: --_--g-pin-mixin-border-radius;
```

Reply:
This will make this variable public, is this really your intention?